lib/coney/connection_server.ex

```diff
@@ -85,29 +140,43 @@ defmodule Coney.ConnectionServer do
 
   defp rabbitmq_connect(
          %State{
-           consumers: consumers,
            adapter: adapter,
            settings: settings,
            topology: topology
          } = state
        ) do
     conn = adapter.open(settings)
     adapter.init_topology(conn, topology)
-    start_consumers(consumers, adapter, conn)
 
```

Comment:
Maybe we should move the Process.monitor from the adapter to here, seems like a more fitting context